src/Servers/HttpSys/src/MessagePump.cs

```diff
@@ -176,14 +176,17 @@ internal void SetShutdownSignal()
         // The awaits will manage stack depth for us.
         private async void ProcessRequestsWorker()
         {
+            // Allocate and accept context per loop and reuse it for all accepts
+            using var acceptContext = new AsyncAcceptContext(Listener);
+
             int workerIndex = Interlocked.Increment(ref _acceptorCounts);
             while (!Stopping && workerIndex <= _maxAccepts)
             {
                 // Receive a request
                 RequestContext requestContext;
                 try
                 {
-                    requestContext = await Listener.AcceptAsync().SupressContext();
+                    requestContext = await Listener.AcceptAsync(acceptContext);
```

Comment:
Confused why this was removed? Is SuppressContext used elsewhere?

Reply:
See https://github.com/dotnet/aspnetcore/pull/28308